game_eggs/rimworld/open_world/egg-rimworld-open-world.json

```diff
@@ -0,0 +1,282 @@
+        "startup": "{\r\n    \"done\": \"Network Line Started\"\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#Create the server directory\r\nmkdir -p \/mnt\/server\r\ncd \/mnt\/server\r\napt update\r\napt install -y liblua5.3-0 lua-socket curl jq unzip\r\n\r\n#Remove any old versions\r\nrm -f LinuxX64.zip\r\n\r\n#DotNet Core Download Location\r\nDOTNET_DOWNLOAD=\"https:\/\/download.visualstudio.microsoft.com\/download\/pr\/046afe25-7b88-48ad-a06c-1c3625115c63\/6814f9ca777bc7e2cb4b373027dcdd76\/dotnet-runtime-3.1.31-linux-x64.tar.gz\"\r\n\r\n#Check for latest release & download URLs\r\nLATEST_JSON=$(curl --silent \"https:\/\/api.github.com\/repos\/TastyLollipop\/OpenWorld\/releases\/latest\")\r\nRELEASES=$(curl --silent \"https:\/\/api.github.com\/repos\/TastyLollipop\/OpenWorld\/releases\")\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    echo \"Using latest Rimworld Open World server version\"\r\n    DOWNLOAD_URL=$(echo ${LATEST_JSON} | jq .assets | jq -r .[].browser_download_url | grep -i LinuxX64)\r\nelse\r\n    DOWNLOAD_URL=\"https:\/\/github.com\/TastyLollipop\/OpenWorld\/releases\/download\/${VERSION}\/LinuxX64.zip\"\r\nfi\r\n\r\n#Download DotNET Core\r\necho -e \"Running curl -sSL ${DOTNET_DOWNLOAD} -o dotnet.tar.gz\"\r\ncurl -sSL ${DOTNET_DOWNLOAD} -o dotnet.tar.gz\r\ntar -xvf dotnet.tar.gz\r\nchmod +x dotnet\r\n\r\n#Download the Rimworld Open World server\r\necho -e \"Running curl -sSL ${DOWNLOAD_URL} -o LinuxX64.zip\"\r\ncurl -sSL ${DOWNLOAD_URL} -o LinuxX64.zip\r\necho -e \"Unzipping LinuxX64.zip\"\r\nunzip -o LinuxX64.zip\r\nchmod +x \"Open World Server\"\r\n\r\necho -e \"Generating server settings\"\r\ncat << EOF > \"Server Settings.txt\"\r\nServer Details -\r\nServer Name: My Server Name\r\nServer Description: My Server Description\r\nServer Local IP: 0.0.0.0\r\nServer Port: 25555\r\nMax Players: 300\r\nAllow Dev Mode: False\r\nUse Whitelist: False\r\nUse Enforced Difficulty: False\r\n\r\n- Mod System Details -\r\nUse Modlist Match: True\r\nUse Modlist Config Match (WIP): False\r\nForce Mod Verification: False\r\n\r\n- Chat System Details -\r\nUse Chat: True\r\nUse Profanity filter: True\r\n\r\n- Wealth System Details -\r\nUse Wealth System: False\r\nWealth Warning Threshold: 10000\r\nWealth Ban Threshold: 100000\r\n\r\n- Idle System Details -\r\nUse Idle System: True\r\nIdle Threshold (days): 7\r\n\r\n- Road System Details -\r\nUse Road System: True\r\nAggressive Road Mode (WIP): False\r\nEOF\r\n\r\necho -e \"Generating World Settings\"\r\ncat << EOF > \"World Settings.txt\"\r\n- World Settings -\r\nGlobe Coverage (0.3, 0.5, 1.0): 0.3\r\nSeed: Seed\r\nOverall Rainfall (0-6): 3\r\nOverall Temperature (0-6): 3\r\nOverall Population (0-6): 3\r\nEOF",
+            "container": "ghcr.io\/parkervcp\/yolks:debian",
```

Comment:
You should only need `ghcr.io/parkervcp/installers:debian` to run the install.

You can also skip adding dotnet and other things during the install.